apiserver/root.go

```diff
@@ -676,7 +675,7 @@ func (r *apiHandler) AuthUnitAgent() bool {
 // AuthOwner returns whether the authenticated user's tag matches the
 // given entity tag.
 func (r *apiHandler) AuthOwner(tag names.Tag) bool {
-	return r.entity.Tag() == tag
+	return r.authInfo.Entity.Tag() == tag
```

Comment:
Can we add a helper entity() method to return `r.authInfo.Entity` to abstract a little the implementation detail; double indirection is non-zero cognitive overhead.

Reply:
Sorry that was a fat finger on my behalf.